src/ps-web-apis.ts

```diff
@@ -83,6 +83,10 @@ export interface WhoamiV1 {
      * @return {WhoamiUserInfo} some relevant userdata
      */
     getUserInfo(): Promise<WhoamiUserInfo>;
+    /**
+     * will request customer pseudo id for currently logged user from consent backend
+     */
+    getCustomerId(additionalParameter?: Map<string, string[]>): Promise<string>;
```

Comment:
mich wundert nur dass es hier keine spezifischen parameter gibt... welche additional parameters gibt es denn?

das typing soll ja eigtl. helfen wenn man das modul nutzt. so frag ich mich was ich da mitgeben kann und warum

Reply:
"clientId" ist so ein pflicht parameter. Ich dachte ich mache mir alle parameters als additional ^^ aber ok ich habe jetzt noch "clientId: string" da hinzugefügt

Reply:
aber was sind denn die additional parameters? ... das ist so als ob du in java zu Object castest und das ist dann die API

Reply:
Schonmal generisch für alle alle kommenden Parameter vorbereitet? ;)

Reply:
Ja ich dachte erst dass wir es generisch machen wollen wie bei doLogin, doRegister und so :D aber egal ich habe nur das "clientId" da gelasen. Erst sollte es passen :D